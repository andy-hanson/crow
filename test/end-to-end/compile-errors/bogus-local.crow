main void() summon
	x = z
	_ nat = x ? 1 : 2
