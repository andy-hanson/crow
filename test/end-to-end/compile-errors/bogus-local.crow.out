exit code: 1
stderr:
test/end-to-end/compile-errors/bogus-local.crow 2:9-2:10 There is no function or variable named 'z'.
Uncaught exception: Error: Reached compile error
	at main
